app/cronjob/trailingTradeHelper/__tests__/queue.test.js

```diff
@@ -53,30 +56,35 @@ describe('queue', () => {
       it('triggers new Queue for BTCUSDT', () => {
         expect(mockQueue).toHaveBeenCalledWith('BTCUSDT', expect.any(String), {
           prefix: `bull`,
-          settings: {
-            guardInterval: 1000
+          limiter: {
+            max: 1,
+            duration: 10000,
```

Comment:
@chrisleekr Isn't 10 seconds too much?

Reply:
@uhliksk 
Oh, is it acting weird?

Reply:
@chrisleekr
It is updating price once per 10 seconds which can skip price spikes easily. For example, if your sell trigger price here is 16897.00, you can skip an opportunity to sell silently just because you'll update before and after the spike.

![image](https://user-images.githubusercontent.com/37454226/209492208-166d6f3f-7b00-4cde-8117-b62f0ccbb453.png)